draft-gpew-priv-ppm.md

```diff
@@ -659,12 +659,18 @@ minutes, to account for clock skew.
 Once a set of clients have uploaded their reports to the leader, the leader can
 send them to the helpers to be verified and aggregated. In order to enable the
 system to handle very large batches of reports, this process can be performed
-incrementally. To aggregate a set of reports, the leader sends a message to each
-helper containing those report shares. The helper then processes them (verifying
-the proofs and incorporating their values into the ongoing aggregate) and replies
-to the leader.
+incrementally. Verification of a set of reports is referred to as an aggregation
+job. Each aggregation job is associated with at most one PPM task, and a PPM
```

Comment:
"at most one" suggests an aggregation job could be associated with zero PPM tasks.
```suggestion
job. Each aggregation job is associated with exactly one PPM task, and a PPM
```